erpnext/e_commerce/website_item_indexing.py

```diff
@@ -0,0 +1,214 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# License: GNU General Public License v3. See license.txt
+
+import frappe
+from frappe.utils.redis_wrapper import RedisWrapper
+
+from redisearch import (
+        Client, AutoCompleter,
+        Suggestion, IndexDefinition, 
+        TextField, TagField
+	)
+
+def is_search_module_loaded():
+    cache = frappe.cache()
+    out = cache.execute_command('MODULE LIST')
+
+    parsed_output = " ".join(
+        (" ".join([s.decode() for s in o if not isinstance(s, int)]) for o in out)
+    )
+
+    return "search" in parsed_output
+
+# Decorator for checking wether Redisearch is there or not
+def redisearch_decorator(function):
```

Comment:
```suggestion
def redisearch(function):
```

The function name should convey the use case. No need to put "decorator" in name.

Reply:
The name should convey the use case +1. That's why just  `redisearch` or `redisearch_decorator` are bad names. Perhaps something along the lines of `is_redisearch_loaded`?